src/ctia/http/handler.clj

```diff
@@ -78,17 +80,23 @@
     (assert services->routes (str "Missing :services->routes for " (:entity entity)))
     (services->routes services-map)))
 
+(defn- entity-enabled?
+  [entity
+   {{:keys [enabled?]} :FeaturesService}]
+  (enabled? entity))
+
 (defmacro entity-routes
   [entities services-map]
   (let [gsm (gensym 'services-map)]
   `(let [~gsm ~services-map]
      (compojure.api.sweet/routes
       ~@(for [entity (remove :no-api?
```

Comment:
this already totally disable routes for some entities.

Reply:
I know that, but I don't see a way of adding the key in runtime. Pls. see the comment: https://github.com/threatgrid/ctia/pull/1017#issuecomment-716940568